CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+
+- The Publisher extension will activate, but not initialize when the VSCode operates in Restricted Mode.
+  Initialization will commence when/if the workspace is trusted (which can be done via a button within the
+  publisher's view).
```

Comment:
Just some word smithing suggestions:

> Improved the extension's behavior in VS Code workspace restricted mode. The extension will activate, but prompt to manage workspace trust so it can initialize.